PinpointKit/PinpointKit/Sources/Core/Editing/EditImageViewController.swift

```diff
@@ -408,7 +413,10 @@ open class EditImageViewController: UIViewController, UIGestureRecognizerDelegat
         let dismissButton = UIBarButtonItem(title: interfaceCustomization?.interfaceText.textEditingDismissButtonTitle, style: .done, target: self, action: #selector(EditImageViewController.endEditingTextViewIfFirstResponder))
         dismissButton.setTitleTextAttributes([NSAttributedStringKey.font: buttonFont], for: UIControlState())
         
-        navigationItem.setRightBarButton(dismissButton, animated: true)
+        let fixedSpace = UIBarButtonItem(barButtonSystemItem: .fixedSpace, target: nil, action: nil)
+        fixedSpace.width = 10
```

Comment:
I feel that we should ensure that the width for the fixed space here matches it in its initial declaration, so could we either create a private constant in this class in a `LayoutConstants` struct, or perhaps even better, a computed property for, say, `fixedSpaceBarButtonItem` and use it in both locations?